### PR TITLE
Get Quic tests running in CI again

### PR DESCRIPTION
**Pull request by wtgodbe**
Going back to the queue that we were using before to see if that gets Quic tests running again.

### CODE REVIEW

**Review by dougbu — APPROVED** [review]
Ick, icky ickness but if it works&hellip;

**Comment by wtgodbe** [comment]
If I confirm this fixes out I'll ask eng/runtime folks for help figuring out why changing the base image would change the `msquic` library being installed, but for now we want to at least get the tests running again.

File: eng/targets/Helix.Common.props [hunk 1]
```diff
@@ -19,7 +19,7 @@
   
   <!-- x64 Quarantined-only (quarantined-pr.yml and quarantined-tests.yml) test queues -->
   <ItemGroup Condition="'$(IsRequiredCheck)' == 'true' AND '$(TargetArchitecture)' == 'x64' AND '$(_UseHelixOpenQueues)' == 'true' AND '$(RunQuarantinedTests)' == 'true'">
-    <HelixAvailableTargetQueue Include="(Fedora.34.Amd64.Open)ubuntu.1804.amd64.open@mcr.microsoft.com/dotnet-buildtools/prereqs:fedora-34-helix-20210728124700-4f64125" Platform="Linux" />
+    <HelixAvailableTargetQueue Include="(Fedora.34.Amd64.Open)ubuntu.1604.amd64.open@mcr.microsoft.com/dotnet-buildtools/prereqs:fedora-34-helix-20210728124700-4f64125" Platform="Linux" />
```
Comment (JamesNK) [inline eng/targets/Helix.Common.props:22]: Changing this accidentally broke QUIC tests once. I think a comment should be added here to call out QUIC requirements so they are considered next time someone changes the queue.

Comment (JamesNK) [reply eng/targets/Helix.Common.props:22]: (maybe add the comment in a follow-up PR once we understand what is going on)

**Comment by wtgodbe** [comment]
Weird, it looks like everything is getting skipped on the Fedora queue, though maybe I'm reading the tests tab wrong? For fedora I see only successful work items, whereas for for other queues I see work items & individual tests. @MattGal is there a way to grab the testresults.xml files from the successful Fedora work items?

**Comment by wtgodbe** [comment]
Never mind, found the console log: https://helixre8s23ayyeko0k025g8.blob.core.windows.net/dotnet-aspnetcore-refs-pull-35669-merge-a1eb5c6024db416aac/Microsoft.AspNetCore.Server.Kestrel.Transport.Quic.Tests--net6.0/1/console.0015b383.log?sv=2019-07-07&se=2021-09-13T20%3A17%3A09Z&sr=c&sp=rl&sig=7Pce3tcT4yFA8eUWsV0wy2OGalpHhZL7wQ4SBj9FpFg%3D

38 of the 40 tests are running, 2 get skipped (`ClientCertificate_Required_Sent_Populated` and `ClientCertificate_Required_NotSent_ConnectionAborted`. They all pass, but the results don't actually get reported. Looks like that's also the case today with the `1804` queue, without this change: https://helixre8s23ayyeko0k025g8.blob.core.windows.net/dotnet-aspnetcore-refs-heads-main-5acd394e1bd1459194/Microsoft.AspNetCore.Server.Kestrel.Transport.Quic.Tests--net6.0/1/console.cee8175d.log?sv=2019-07-07&se=2021-09-13T20%3A32%3A43Z&sr=c&sp=rl&sig=Y69EMVlXqkUOcJwryFYXE8o%2FKjiaq9%2BH2rCONXTdR2k%3D

So the issue isn't that the tests aren't running, it's that they aren't reporting results for that queue for some reason. The same appears to be true for _all_ tests in that queue. @MattGal any idea why that might be happening or how we can investigate?

**Comment by MattGal** [comment]
>  @MattGal any idea why that might be happening or how we can investigate?

Taking a peek

**Comment by MattGal** [comment]
The non-console log should, but does not contain something like this:
```
2021-08-19T22:47:49.566Z	INFO   	packing_test_reporter(43)	unpack_results	Unpacking 35926 bytes from 'D:\h\w\B3340978\w\AC7609D5\e\__test_report.json'
2021-08-19T22:47:49.597Z	INFO   	packing_test_reporter(50)	unpack_results	Unpacked 88 test reports
2021-08-19T22:47:49.613Z	INFO   	executor(190)	run	Local retry count reached, no re-execution attempted
2021-08-19T22:47:50.695Z	INFO   	azure_devops_result_publisher(111)	_process	uploaded 88 results
```
... so reporting just didn't happen. Tagging @ChadNedzlek to investigate.

**Comment by MattGal** [comment]
@wtgodbe we had a bug and this investigation found a gap in testing.  Test runs started after now should successfully report results;  let me know if you observer otherwise.

**Comment by wtgodbe** [comment]
Nice! Just kicked off a job to test: https://dev.azure.com/dnceng/public/_build/results?buildId=1317761&view=results

**Comment by dougbu** [comment]
@wtgodbe if the test succeeds, will we need to downgrade the base Ubuntu image❔

**Comment by wtgodbe** [comment]
> @wtgodbe if the test succeeds, will we need to downgrade the base Ubuntu image❔

No, the tests run just fine on 1804, it was just a coincidence that they stopped reporting around the time we switched to it (we think)

**Comment by dougbu** [comment]
Nice that we may not need this PR 🥇

**Comment by wtgodbe** [comment]
I'll close this if/when https://dev.azure.com/dnceng/public/_build/results?buildId=1317761&view=results gives us Quic test results

**Pull request closed without merging**